quinn-proto/src/connection/mod.rs

```diff
@@ -1399,6 +1429,25 @@ impl Connection {
                 // Notify ack frequency that a packet was acked, because it might contain an ACK_FREQUENCY frame
                 self.ack_frequency.on_acked(packet);
 
+                if let Some(timestamp_iter) = timestamp_iter.as_mut() {
+                    while let Some(v) = timestamp_iter.peek() {
```

Comment:
Why do we need/want to `peek()` here?

Reply:
No longer needed since the algorithm was reworked.